src/common/actions/repositories.js

```diff
@@ -67,9 +68,15 @@ export function fetchUserRepositories(pageNumber) {
           if (result.pageLinks) {
             dispatch(setPageLinks(result.pageLinks));
           }
+        })
+        .catch(error => {
+          return Promise.reject(error);
         });
       })
-      .catch(error => dispatch(fetchRepositoriesError(error)));
+      .catch(error => {
+        log.warn(error);
```

Comment:
@bartaz:
<img width="770" alt="screen shot 2017-02-08 at 09 42 04" src="https://cloud.githubusercontent.com/assets/1410204/22731537/e741f3ca-ede2-11e6-93ed-c91ba05d1577.png">

If I don't log the error out, this fails silently. It's also another story for another day about why **this** error is being thrown **here**, but I have limited time to investigate this right now.